src/client/date.formatter.ts

```diff
@@ -1,7 +1,7 @@
 import moment = require("moment");
 
 const DISPLAY_DATE_FORMAT: string = "D MMMM YYYY";
-const REASON_DATE_FORMAT: string = "YYYY-MM-DD";
+const REASON_DATE_FORMAT: string = "YYYY-MM-DDTHH:mm:ss";
```

Comment:
Again, constant name should reflect that when using it to format a string, a time component is expected/will be output. REASON_DATE_TIME_FORMAT maybe?

Reply:
Done